components/x11/xcolor/Makefile

```diff
@@ -13,45 +13,36 @@
 # Copyright 2015 Aurelien Larcher
 #
 
+BUILD_BITS=64
+BUILD_STYLE=justmake
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=           xcolor
 COMPONENT_VERSION=        7.5
+COMPONENT_REVISION=       1
 COMPONENT_FMRI=           x11/xcolor
 COMPONENT_CLASSIFICATION= System/X11
 COMPONENT_SUMMARY=  xcolor - colormap display program
 COMPONENT_SRC=      src
 COMPONENT_PROJECT_URL = https://hg.java.net/hg/solaris-x11~x-s12-clone
 COMPONENT_LICENSE=      Oracle
-COMPONENT_LICENSE_FILE= xcolor.license
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/justmake.mk
-include $(WS_TOP)/make-rules/ips.mk
+TEST_TARGET=$(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
 
-PATH=/usr/gnu/bin:/usr/bin
+PATH=$(USRBINDIR)
```

Comment:
Is /usr/gnu/bin not needed anymore?

Reply:
Just remove PATH= line?

Reply:
Yes it does not seem to be used, I just removed it. Thank you :)

Reply:
Hm, I meant it the other way around: why did you leave out /usr/gnu/bin from PATH :)

Reply:
OK OK, it does not seems to be used but re-added it as before anyways.